### ISSUE TITLE
Copy file to clipboard

### ISSUE THREAD

**Issue by astoilkov** [gildas-lormeau/SingleFile#917]
**Is your feature request related to a problem? Please describe.**
I want to use it in my knowledge base. Now it's hard because I need to go to Finder and copy the saved file to the clipboard and then be able to paste it into my notes app.

**Describe the solution you'd like**
I want to click the SingleFile button and then hit ⌘V in my notes (my notes app copies the file in a /attachments folder and links to it).

**Describe alternatives you've considered**
Using the CLI app and doing a Raycast or Alfred command that will call SingleFile and copy the file to the clipboard.

**Additional context**
I'm aware of the Copy to clipboard functionality but that doesn't work because it copies the text of the file and I need the actual file (what happens when you hit ⌘C in Finder).

**Comment by gildas-lormeau** [comment]
I don't think this is technically possible within web extensions. Maybe this could be delegated to SingleFile Companion though.

**Comment by astoilkov** [comment]
Do you mean a new option in Companion or this is already possible? I've read the Companion docs but I am not sure if there is a way to do it with the current implementation.

**Comment by gildas-lormeau** [comment]
No it's not possible today with SingleFile Companuion. Actually, I took some time to verify if it would be possible to code this feature in SingleFile Companion but it looks like it's also impossible. Generally speaking, I am not familiar with how copy/paste works on operating systems in details. However, I have the impression that in order to copy/paste a file, it must first exist on the file system, which is not really practical.

**Comment by astoilkov** [comment]
Yes, it's probably too complicated. On macOS, you have to write to `NSFilenamesPboardType` (or `public.file-url`), on Windows `FileNameW`, on Linux I don't even know. I don't know if you even have these permissions.

If this can't be done, feel free to close this issue. Thanks for the time spend on this!
